.github/workflows/bdd.yml

```diff
@@ -0,0 +1,62 @@
+name: BDD CI
+on:
+  workflow_call:
```

Comment:
Why we use on.`workflow_call`? and not `pull_request` or `push`.. on other files i see we trigger some on `push`